src/content/ui-api/kubectl-gs/template-cluster.md

```diff
@@ -66,7 +76,7 @@ For OpenStack (`--provider openstack`):
 {{< /tab >}}
 {{< /tabs >}}
 
-**Note:** With the exception of OpenStack, the CRs generated by this command won't trigger the creation of any worker nodes. Please see the [template nodepool]({{< relref "/ui-api/kubectl-gs/template-nodepool" >}}) for instructions on how to create worker node pools on all other providers.
+**Note:** With the exception of OpenStack and GCP, the CRs generated by this command won't trigger the creation of any worker nodes. Please see the [template nodepool]({{< relref "/ui-api/kubectl-gs/template-nodepool" >}}) for instructions on how to create worker node pools on all other providers.
```

Comment:
On `gcp` and `openstack`, one nodepool is created by default right? We might as well say that.

Reply:
Done. Just check if the wording is okay